upgrading/topics/keycloak/changes.adoc

```diff
@@ -11,14 +11,16 @@ migration is finished.
 
 ===== Protocol mapper SPI addition
 Related to this, there is a small addition in the (unsupported) Protocol Mappers SPI. You can be affected only if you
-implemented a custom ProtocolMapper. There is a new `getPriority()`` method on the ProtocolMapper interface. The method has the
+implemented a custom ProtocolMapper. There is a new `getPriority()` method on the ProtocolMapper interface. The method has the
 default implementation set to return 0. If your protocol mapper implementation relies on the roles in the access token `realmAccess`
 or `resourceAccess` properties, you may need to increase the priority of your mapper.
 
 ===== Audience resolving
 
 Audiences of all the clients, for which authenticated user has at least one client role in the token, are automatically added
-to the `aud` claim in the access token now.
+to the `aud` claim in the access token now. On the other hand, access token may not automatically contains the audience of the
+frontend client, for which it was issued for. You should follow the link:{adminguide_link}#_audience[{adminguide_name}]
+for more details.
 
```

Comment:
...to the `aud` claim in the access token now. On the other hand, an access token may not automatically contain the audience of the frontend client, for which it was issued. Read the link:{adminguide_link}#_audience[{adminguide_name}] for more details.